kwok/cloudprovider/helpers.go

```diff
@@ -39,12 +39,20 @@ var (
 	familyDelim = regexp.MustCompile(`[.-]`)
 )
 
+type KwokOfferings []KwokOffering
```

Comment:
Does it make sense to decouple this offering update change from the update to the E2E tests and actions so that we can merge those changes quicker since they were already reviewed?

Reply:
Since we added requirements in offerings, this won't work without it. That being said, I can split into another PR if you want.

Reply:
Might be nice to split it out so that we don't hold the utils changes on merging due to comments and back-and-forth with these changes